### PR TITLE
Loaders: Improve code quality

### PR DESCRIPTION
**Pull request by Mugen87**
This PR improves the code quality of most loaders. So minor things like better code style, added missing semicolons and correct spaces/indentations (according to [Mr.doob's Code Style™](https://github.com/mrdoob/three.js/wiki/Mr.doob%27s-Code-Style%E2%84%A2)).

### CODE REVIEW

**Comment by mrdoob** [comment]
Many thanks!

File: examples/js/loaders/FBXLoader.js [hunk 1]
```diff
@@ -4061,7 +4065,6 @@
 			// The first three data sizes depends on version.
 			var endOffset = ( version >= 7500 ) ? reader.getUint64() : reader.getUint32();
 			var numProperties = ( version >= 7500 ) ? reader.getUint64() : reader.getUint32();
-			var propertyListLen = ( version >= 7500 ) ? reader.getUint64() : reader.getUint32();
```
Comment (takahirox) [inline examples/js/loaders/FBXLoader.js]: Why did you remove this line?
I don't think we should remove...

Comment (takahirox) [reply examples/js/loaders/FBXLoader.js]: Ah, probably you did that because `propertyListLen` is unused, correct?
But the line is necessary to move the pointer forward in the binary file.

Comment (mrdoob) [reply examples/js/loaders/FBXLoader.js]: Uh oh! I'll revert.

Comment (Mugen87) [reply examples/js/loaders/FBXLoader.js]: > But the line is necessary to move the pointer forward in the binary file.

Sry guys! I've overlooked that...

Comment (mrdoob) [reply examples/js/loaders/FBXLoader.js]: No worries! 😊
